job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/Constants.groovy

```diff
@@ -23,6 +23,10 @@ class Constants {
     static final String BRANCH = "master"
     static final String GITHUB_ORG_UNIT = "kiegroup"
     static final String PULL_REQUEST_FOLDER = "pullrequest"
+    static final String FDB_FOLDER = "fullDownstream"
+    static final String CDB_FOLDER = "compileDownstream"
```

Comment:
`compile` would probably be a more appropriated name

Reply:
it is true we have only one CDB, but still it should be meaningfull for first view, so just **compile** won't show that IMO